tests/kube_test.py

```diff
@@ -0,0 +1,167 @@
+from click.testing import CliRunner
+from utilities_common.db import Db
+
+show_server_output_0="""\
```

Comment:
i think we should follow the same cli output format as others.

kubernetes master

ip    insecure   disable   port
---   ---------   --------   -----
a.b.c.d true false 6443

Reply:
ok.

Reply:
```
admin@sonic:~$ show kube server config
ip           port    insecure    disable
-----------  ------  ----------  ---------
10.3.157.24  6443    True        False
```